lxd/db/patches.go

```diff
@@ -1,16 +1,13 @@
 package db
 
-import (
-	"database/sql"
-	"fmt"
-)
+import "fmt"
```

Comment:
Another one of those that I'd prefer to keep with the long formatting.

Reply:
Fixed.